object_detection/scripts/object_detection_node.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import rospy
+
+# Import msg types
+from darknet_ros_msgs.msg import BoundingBoxes, BoundingBox
+from sensor_msgs.msg import Image
+from vortex_msgs.msg import BBox, BBoxes
+from std_msgs.msg import Int64, Float64
+from geometry_msgs.msg import PointStamped, Point
+import tf
+
+# Import classes
+from size_estimator import SizeEstimatorClass as SEC
+from coord_pos import CoordPositionClass as CPC
+
+class OD_NODE():
+    test = 'T'
```

Comment:
To stay consistent with the rest of your class naming, this should be ObjectDetectionNode